--- origsrc/PyX-0.10/setup.cfg	2006-04-28 05:45:09.000000000 -0500
+++ src/PyX-0.10/setup.cfg	2007-10-07 00:04:46.984375000 -0500
@@ -2,11 +2,11 @@
 # In this section you can specify which c extension modules should be built
 
 # C extension module for fast t1font decoding and encoding
-build_t1code=0
+build_t1code=1
 
 # Python bindings for the kpathsea library. You need the kpathsea header
 # and library and you may need to specify their location below.
-build_pykpathsea=0
+build_pykpathsea=1
 
 [build_ext]
 # additional include and library directories:
